sdk/identity/identity/src/msal/nodeFlows/nodeCommon.ts

```diff
@@ -63,11 +63,18 @@ export abstract class MsalNode extends MsalBaseUtilities implements MsalFlow {
       ...options.tokenCredentialOptions,
       authorityHost
     });
+
+    let clientCapabilities: string[] = ["CP1"];
+    if (process.env.AZURE_IDENTITY_DISABLE_CP1) {
+      clientCapabilities = [];
+    }
+
     return {
       auth: {
         clientId,
         authority: authorityHost,
-        knownAuthorities: getKnownAuthorities(tenantId, authorityHost)
+        knownAuthorities: getKnownAuthorities(tenantId, authorityHost),
+        clientCapabilities: clientCapabilities as any // https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/3676
```

Comment:
We have to do some type-casting to bypass this issue: https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/3676

However, this means we have a lint issue.

Reply:
Samuel Kubai has a PR: https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/3678